$MirOS$

	 mpcc -DG_DISABLE_DEPRECATED -O2 -pipe -std=gnu99 -Os -fweb -frename-registers -Wformat -Wstrict-aliasing -Wbounded -fwrapv -fno-align-functions -fno-align-labels -falign-loops=4 -falign-jumps=4 -march=i486 -mpush-args -mpreferred-stack-boundary=2 -fhonour-copts -Wall -c composite_line_22_4a4_mmx.S  -fPIC -DPIC -o .libs/composite_line_22_4a4_mmx.o
	composite_line_22_4a4_mmx.S:192:3: invalid preprocessing directive #%
	composite_line_22_4a4_mmx.S:193:3: invalid preprocessing directive #pxor

--- gdk-pixbuf/pixops/composite_line_22_4a4_mmx.S.orig	Mon Jul 16 19:44:44 2007
+++ gdk-pixbuf/pixops/composite_line_22_4a4_mmx.S	Tue Dec  9 22:08:38 2008
@@ -189,8 +189,6 @@ __pixops_composite_line_22_4a4_mmx:
 
 	shll $2, %edx
 
-#	%mm4 will always be already clear here	
-#	pxor %mm4, %mm4
 
 	movl 16(%ebp),%edi
 	movl (%edi, %edx), %eax
